runtime/kusama/src/lib.rs

```diff
@@ -1495,7 +1480,10 @@ mod benches {
 		[pallet_offences, OffencesBench::<Runtime>]
 		[pallet_preimage, Preimage]
 		[pallet_proxy, Proxy]
+		[pallet_ranked_collective, FellowshipCollective]
 		[pallet_recovery, Recovery]
+		[pallet_referenda, Referenda]
+		[pallet_referenda, FellowshipReferenda]
```

Comment:
This results in the pallet being listed twice with `benchmark pallet --list --chain kusama-dev`.  
I think we can append the instance to the pallet name in the benchmarking CLI.  
What do you think?

Reply:
yes, we need some better solution to handle instances. not sure exactly what that is, but it should show up in the list, and be targeted with the appropriate command, and have a unique output